inference-api/python_client/client.py

```diff
@@ -0,0 +1,112 @@
+import os
+import requests
```

Comment:
`requests`, `asgiref`, and `retry` are all third party libraries. In this repo I've been following the same style as in our other repos where we separate imports by first and third party imports.

Reply:
Modified to be consistent with convention